src/main/java/model/data/DataTableBuilder.java

```diff
@@ -60,7 +70,7 @@ public void addRow(DataRow row) {
 	 * @return the constructed DataColumn
 	 */
 	public DataColumn createColumn(String name, Class<? extends DataValue> type) {
-		return new DataColumn(name, type);
```

Comment:
Op zich, mag createColumn nu wel gelijk die kolom toevoegen, want dat werkt chiller en we hebben nog geen geval waarin je hem wel aan wil maken, maar niet toevoegen.